DfE.FindInformationAcademiesTrusts/Pages/Trusts/Contacts.cshtml.cs

```diff
@@ -1,10 +1,36 @@
 using DfE.FindInformationAcademiesTrusts.Data;
+using Microsoft.AspNetCore.Mvc;
 
 namespace DfE.FindInformationAcademiesTrusts.Pages.Trusts;
 
 public class ContactsModel : TrustsAreaModel
 {
+    public Governor? ChairOfTrustees { get; set; }
+
+    public Governor? AccountingOfficer { get; set; }
+
+    public Governor? ChiefFinancialOfficer { get; set; }
+
+    public string ContactNameNotAvailableMessage = "No contact name available";
```

Comment:
Can we make these strings `const`? This means that they will be computed at compile time rather than runtime and will give a (tiny) performance boost (which is why it's good practice in .NET to use consts where possible)